app-web/abdownloadmanager/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=ab-download-manager
+PKGSEC=web
+PKGDES="A multifunctional file downloader"
+BUILDDEP="openjdk"
+PKGDEP="openjdk"
```

Comment:
There are multiple `.so` files in the runtime directories, please investigate what their dependencies are.